src/confcom/azext_confcom/docs/v1_json_file_format.md

```diff
@@ -0,0 +1,221 @@
+| `scenario`   | string  | No                                            | Adds appropriate mounts and environment variables for different deployment scenarios. If deploying to `vn2`, this field is required |
+| `fragments`  | array   | Yes (either `containers` or `fragments` must be defined) | Specifies policy fragments to include                                                          |
+| `containers` | array   | Yes (either `containers` or `fragments` must be defined) | Container configurations to deploy                                                             |
+
+## Detailed Field Reference
+
+### Top-Level Fields
+
+#### `version`
+
+- **Type**: String
+- **Required**: Yes
+- **Description**: Version identifier for the output policy format
+- **Allowed Values**: "1.0"
+- **Example**: `"version": "1.0"`
+
+#### `scenario`
+
+- **Type**: String
+- **Required**: No
+- **Description**: Adds appropriate mounts and environment variables for different deployment scenarios.
+If deploying to `vn2`, this field is required.
+The default value is `aci`
+- **Allowed Values**: "vn2", "aci"
+- **Example**: `"scenario": "vn2"`
+
+#### `fragments`
+
+- **Type**: Array of objects
+- **Required**: Yes (either `containers` or `fragments` must be defined)
+- **Description**: Policy fragments that should be included and imported when enforcing the security policy
+
+#### `containers`
+
+- **Type**: Array of objects
+- **Required**: Yes (either `containers` or `fragments` must be defined)
+- **Description**: Container specifications for deployment
+
+### Fragment Objects
+
+These objects define all the information necessary for importing and using a fragment.
+
+#### fragments
+
+- **Type**: Array of objects
+- **Required**: Yes (can be empty)
+- **Description**: Defines reusable policy fragments that can be referenced and imported into container policies.
+
+Each fragment object must include the following fields:
+
+| Field         | Type   | Required | Description                                                          |
+|---------------|--------|----------|----------------------------------------------------------------------|
+| `issuer`      | string | Yes      | DID identifier of the fragment signer                                |
+| `feed`        | string | Yes      | Name for the fragment. Can be its address in a container registry (for standalone fragments) or an arbitrary name (for image-attached fragments)                                             |
+| `minimum_svn` | string | Yes      | Minimum Security Version Number required                             |
+| `includes`    | array  | Yes      | Variables included in the fragment (e.g., "containers", "fragments") |
+
+**Example**:
+
+```json
+{
+  "issuer": "did:x509:0:sha256:I__iuL25oXEVFdTP_aBLx_eT1RPHbCQ_ECBQfYZpt9s::eku:1.3.6.1.4.1.311.76.59.1.3",
+  "feed": "contoso.azurecr.io/infra",
+  "minimum_svn": "1",
+  "includes": ["containers"]
+}
+```
+
+### containers
```

Comment:
Put this in the section on line 45 and add an example of a full container config